recipes/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: cobra-meta
+  version: 1.2.2
+
+source:
+  url: https://github.com/linxingchen/cobra/archive/refs/tags/v1.2.2.tar.gz
```

Comment:
Please template the name and the version out.

Reply:
Done. please check. thank you.